modules/cloud-sql/variables.tf

```
@@ -214,11 +244,11 @@ variable "resource_timeout" {
 # See https://github.com/hashicorp/terraform/issues/1178 for more details.
 # This can be used to make sure the module resources are created after other bootstrapping resources have been created.
 # For example:
-# dependencies = ["${google_service_networking_connection.private_vpc_connection.network}"]
+# dependencies = [google_service_networking_connection.private_vpc_connection.network]
 # ---------------------------------------------------------------------------------------------------------------------
 
 variable "dependencies" {
   description = "Create a dependency between the resources in this module to the interpolated values in this list (and thus the source resources). In other words, the resources in this module will now depend on the resources backing the values in this list such that those resources need to be created before the resources in this module, and the resources in this module need to be destroyed before the resources in the list."
-  type        = "list"
+  type        = list(any)
```

This should be
```suggestion
  type        = list(string)
```

or `any` (without `list`). `list(any)` requires all the elements to have the same type, so it is much more error prone.

---

changed to `list(string)`